capnpy/compiler/request.py

```diff
@@ -77,7 +82,8 @@ def emit(self, m):
             m.w('from %s import __compiler, __schema__' % m.tmpname)
         #
         m.w('__capnpy_version__ = {version!r}', version=capnpy.__version__)
-        m.w('_check_version(__capnpy_version__)')
+        if m.version_check:
+            m.w('_check_version(__capnpy_version__)')
```

Comment:
Added it. However, if you think that warning people about not version checked schemas is important, maybe a bigger warning on the top of the file should be added as well / instead.